tests/fuzz/corpora/basewords.txt

```diff
@@ -0,0 +1,1468 @@
+Frank
+kennt
+Habt
+verletzt
+Nachricht
+vielen
+Willkommen
+hart
+lasse
+außer
+Gefängnis
+Schnell
+Sicherheit
+wem
+ehrlich
+interessiert
+sagten
+Sex
+ziehen
+Welche
+Augenblick
+Klasse
+Bild
+Dies
+stark
+Eure
+zehn
+Sinn
+David
+vorstellen
+Tom
+Traum
+länger
+manchmal
+sechs
+sollst
+gestorben
+Tages
+war's
+Kraft
+verschwinden
+gesprochen
+heraus
+genauso
+wärst
+Eins
+damals
+heißen
+Liebling
+wahrscheinlich
+deshalb
+Stelle
+halt
+lhren
+Bill
+erwartet
+solche
+meisten
+bringe
+still
+umgebracht
+Stimme
+General
+Entschuldige
+rede
+Vergiss
+Bewegung
+führen
+höre
+Sonst
+Geschäft
+werdet
+konnten
+wartet
+Raus
+Habe
+hinten
+fällt
+wen
+Sekunden
+Manchmal
+gerettet
+Naja
+jemals
+Meinung
+geschehen
+schlimm
+Eigentlich
+findet
+gewinnen
+Scheiß
+Gedanken
+erinnere
+George
+Herren
+mögen
+Earl
+York
+klingt
+vorher
+Auge
+hab'
+setzen
+cool
+Anfang
+danach
+Lebens
+Monate
+Beste
+benutzen
+Nehmen
+fühlen
+Ärger
+Rolle
+gefährlich
+Dort
+gelernt
+fallen
+böse
+werd
+Nähe
+geschickt
+liebt
+Tisch
+Raum
+Ziel
+jedem
+fliegen
+Wohnung
+Dafür
+lhnen
+erfahren
+dumm
+eher
+vorsichtig
+herum
+Klappe
+Laden
+schicken
+aufs
+großer
+Warten
+Charlie
+Euer
+richtige
+de
+la
+le
+et
+les
+des
+en
+un
+une
+que
+est
+pour
+qui
+dans
+par
+plus
+pas
+au
+sur
+ne
+se
+Le
+ce
+il
+sont
+La
+Les
+ou
+avec
+son
+Il
+aux
+d'un
+En
+cette
+d'une
+ont
+ses
+mais
+comme
+tout
+nous
+sa
+Mais
+fait
+été
+aussi
+leur
+bien
+peut
+ces
+y
+deux
+A
+ans
+l
+encore
+n'est
+marché
+d
+Pour
+donc
+cours
+qu'il
+moins
+sans
+C'est
+Et
+si
+entre
+Un
+Ce
+faire
+elle
+c'est
+peu
+vous
+Une
+prix
+On
+dont
+lui
+également
+Dans
+effet
+pays
+cas
+De
+millions
+Belgique
+BEF
+mois
+leurs
+taux
+années
+temps
+groupe
+bois
+meilleure
+l'argent
+perspectives
+développer
+celui-ci
+oeuvre
+structure
+suivre
+tiers
+prise
+professionnels
+raisons
+néanmoins
+preuve
+social
+bénéficiaire
+couleurs
+mondial
+Cet
+maintenant
+essentiellement
+prévu
+Japon
+prévisions
+centrale
+Alors
+international
+yeux
+PME
+l'a
+ait
+bonnes
+opérations
+pied
+l'art
+pourraient
+Londres
+juge
+devra
+uniquement
+corps
+divers
+Parmi
+numéro
+réduire
+Tous
+texte
+tenu
+budget
+l'étranger
+pression
+mes
+n'était
+style
+économiques
+Jacques
+montre
+population
+analystes
+S
+processus
+placement
+classique
+dividende
+rester
+publics
+fortement
+plein
+wallonne
+DEM
+Express
+faudra
+travailler
+Crédit
+directement
+prime
+Flandre
+crédit
+monnaie
+précise
+appel
+Autre
+travaux
+l'occasion
+juste
+Chaque
+put
+tableau
+terre
+permettent
+devenu
+rouge
+mémoire
+partenaires
+rapide
+travailleurs
+joue
+objectif
+salle
+parle
+musique
+milieu
+d'entreprise
+autorités
+chute
+régime
+d'autant
+liste
+opération
+bout
+performances
+électronique
+haute
+responsables
+lancé
+voitures
+patron
+Malgré
+affiche
+situe
+B
+l'image
+études
+Microsoft
+condition
+retrouve
+Aux
+revient
+Belgacom
+route
+Ensuite
+Luxembourg
+campagne
+comptes
+hors
+culture
+Commission
+d'entre
+possibilités
+semestre
```

Comment:
__[Misspell]__

"semestre" is a misspelling of "semester"


<!-- Comment by [Sider](https://sider.review) -->